diff --git a/node_modules/openvidu-browser/lib/OpenViduInternal/ScreenSharing/Screen-Capturing.js b/node_modules/openvidu-browser/lib/OpenViduInternal/ScreenSharing/Screen-Capturing.js
index c565aaf..fdc1552 100644
--- a/node_modules/openvidu-browser/lib/OpenViduInternal/ScreenSharing/Screen-Capturing.js
+++ b/node_modules/openvidu-browser/lib/OpenViduInternal/ScreenSharing/Screen-Capturing.js
@@ -2,14 +2,16 @@ var chromeMediaSource = 'screen';
 var sourceId;
 var screenCallback;
 var isFirefox = typeof window.InstallTrigger !== 'undefined';
-var isOpera = !!window.opera || navigator.userAgent.indexOf(' OPR/') >= 0;
+var isOpera = !!window.opera || (navigator.userAgent && navigator.userAgent.indexOf(' OPR/') >= 0);
 var isChrome = !!window.chrome && !isOpera;
-window.addEventListener('message', function (event) {
-    if (event.origin != window.location.origin) {
-        return;
-    }
-    onMessageCallback(event.data);
-});
+if(window.addEventListener){
+    window.addEventListener('message', function (event) {
+        if (event.origin != window.location.origin) {
+            return;
+        }
+        onMessageCallback(event.data);
+    });
+}
 function onMessageCallback(data) {
     if (data == 'PermissionDeniedError') {
         if (screenCallback)
